deps/chakrashim/tools/update-changelog.js

```diff
@@ -179,6 +185,15 @@ function pipe(proc, onLine) {
   return proc;
 }
 
+function command_sync(command, args) {
+  var x = child_process.spawnSync(command, args);
+  return x.status == 0 ? x.stdout.toString(): x.stderr.toString();
```

Comment:
In case of error, `error` object of child process could be also useful. `x.error.toString() + "\n" + x.stderrr;`